lib/views/pages/join_organization.dart

```diff
@@ -40,7 +41,7 @@ class _JoinOrganizationState extends State<JoinOrganization> {
     } else if (!result.hasException && !result.loading) {
        _successToast("Sucess!");
       print(result.data);
-      Future.delayed(const Duration(milliseconds: 2000), () {
+      Future.delayed(const Duration(milliseconds: 1000), () {
```

Comment:
same question about the artificial delays

Reply:
same for this...